app/screens/ARViewContainer.js

```diff
@@ -0,0 +1,15 @@
+import { connect } from 'react-redux';
```

Comment:
For example: this file is repeated 3 times, where as before it was done once, and did the same job.